CONTRIBUTING_EN.md

```diff
@@ -156,33 +225,301 @@ Run all the tests with `make test` and make sure that they all passed.
+
+- **Bad**: Fix and improve CSS and HTML issues in the mini pipeline graph and builds dropdown.
+- **Good**: Fix tooltips and hover states in the mini pipeline graph and builds dropdown.
+
+The first example is too focused on implementation details. Users don’t care that we changed CSS and HTML, they care about the final result of those changes.
+
+- **Bad**: Remove null values in the Commit object array returned by find_commits_by_message_with_elastic.
+- **Good**: Fix 500 errors caused by Elasticsearch results referencing commits already collected by the garbage collector.
+
+The first example focuses on how we fixed something, not on what was fixed. The rewritten version clearly describes the final benefit to the user (fewer 500 errors) and when it happens (when searching for commits with Elasticsearch).
+
+Use your best judgment and try to put yourself in the position of someone reading the compiled changelog. Does this entry add value? Does it provide context on where and why the change was made?
+
+#### How to add an entry to the changelog
+
+The changelog is available in the [CHANGELOG.md][changelog] file.
+
+First, you need to identify the type of your change. Types of changes:
+
+- Added for new features. 
+- Changed for changes to existing features. 
+- Deprecated for features that will soon be removed. 
+- Fixed for any bug fixes. 
+- Removed for features that were removed. 
+- Security in case of vulnerabilities.
+
+You should always add new entries to the changelog in the Unreleased section. At the time of release, we will move the changes from the Unreleased section to a new version section.
+
+So, within the Unreleased section, you should add your entry to the appropriate section by type. If there is no section yet for the type of your change, you should add one.
+
+Let’s see some examples. Suppose you have a new Fixed change to add, and the current CHANGELOG.md file looks like this:
+
+```md
+## [Unreleased]
+### Added
+- Utility `get_address_from_cep` [#358](https://github.com/brazilian-utils/brutils-python/pull/358)
+
+### Changed
+- Utility `fmt_voter_id` renamed to `format_voter_id` [#221](https://github.com/brazilian-utils/brutils-python/issues/221)
+```
+
+You would need to add a new Fixed section and include your new entry there:
+
+```md
+## [Unreleased]
+### Added
+- Utility `get_address_from_cep` [#358](https://github.com/brazilian-utils/brutils-python/pull/358)
+
+### Changed
+- Utility `fmt_voter_id` renamed to `format_voter_id` [#221](https://github.com/brazilian-utils/brutils-python/issues/221)
+
+### Fixed
+- My changelog message here. [#<issue_number>](<issue_link>)
+```
+
+Note that the order of sections by type matters. We have a lint that checks this, so the sections must be ordered alphabetically. First Added, then Changed, third Deprecated, and so on.
+
+Now, let’s say you have another entry to add, and its type is Added. Since we already have a section for that, you should just add a new line:
```

Comment:
```suggestion
Now, let’s say you have another entry to add, and its type is `Added`. Since we already have a section for that, you should just add a new line:
```